packages/jest-cli/src/cli/index.js

```diff
@@ -137,8 +137,11 @@ const readResultsAndExit = (
   globalConfig: GlobalConfig,
 ) => {
   const code = !result || result.success ? 0 : globalConfig.testFailureExitCode;
-
-  process.on('exit', () => (process.exitCode = code));
+  
```

Comment:
You have some whitespace on this line, that's why the lint is failing:

```suggestion

```

Reply:
Derp. Will fix.